src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonResource.kt

```diff
@@ -45,17 +46,18 @@ class PrisonResource(private val sentenceEnvelopeService: SentenceEnvelopeServic
   )
   @Operation(summary = "Details of the active sentence envelope, a combination of the person information, the active booking and calculable sentences at a particular establishment (paged response)")
   @PreAuthorize("hasRole('RELEASE_DATE_MANUAL_COMPARER')")
-  @GetMapping("/{establishmentId}/booking/latest/paged/calculable-sentence-envelope")
+  @VerifyAgencyAccess(overrideRoles = ["VIEW_PRISONER_DATA"])
```

Comment:
Odd one.  When I removed the unnecessary calls to the booking service the tests fell over as they were relying on the side effect of getting the booking again.